physical/consul.go

```diff
@@ -30,22 +30,24 @@ const (
 	// be executed too close to the TTL check timeout
 	checkMinBuffer = 100 * time.Millisecond
 
+	// consulRetryInterval specifies the retry duration to use when an
+	// API call to the Consul agent fails.
+	consulRetryInterval = 1 * time.Second
+
 	// defaultCheckTimeout changes the timeout of TTL checks
 	defaultCheckTimeout = 5 * time.Second
 
-	// defaultCheckInterval specifies the default interval used to send
-	// checks
-	defaultCheckInterval = 4 * time.Second
-
 	// defaultServiceName is the default Consul service name used when
 	// advertising a Vault instance.
 	defaultServiceName = "vault"
 
-	// registrationRetryInterval specifies the retry duration to use when
-	// a registration to the Consul agent fails.
-	registrationRetryInterval = 1 * time.Second
+	// reconcileTimeout is how often Vault should query Consul to detect
+	// and fix any state drift.
+	reconcileTimeout = 60 * time.Second
 )
 
+type notifyEvent struct{}
```

Comment:
It certainly can be reduced down to that.  Relying on an explicit type makes it easier to change in the future if we wanted to.  At one point this was sending state down the notification channel, but now it's just an empty signal.  I doubt this will ever be changed back, but from a clarity perspective I'm biased towards the encapsulated explicitness of this vs relying strictly on idiom.